### ISSUE TITLE
Keyboard Navigation (Accessibility)

### ISSUE THREAD

**Issue by mylesblaney** [abelardopardo/ontask_b#189]
Confirm whether all content is accessible through a keyboard interface and whether it requires specific timings for keystrokes? I.e. is it possible to navigate around and complete the task without ever using the mouse?

Whilst it appears possible to navigate around the site by keyboard only – it can be very difficult to see where you have navigated too. Sometimes there is a dotted outline but when there is a button with a black outline this becomes invisible and some of the icons have almost a white outline which likewise is very difficult to see.

**Comment by mylesblaney** [comment]
Consider using a clear outline (yellow block is best) consistently to indicate to users where they have tabbed to make it easier for users to tell where they are.

**Comment by firewuff** [comment]
Yellow has problems with contrast and some colour blindness types if you are concerned with accessibility light blue within the WCAG contrast guidelines is pretty much the standard

**Comment by abelardopardo** [comment]
Any pointers on how to control the color of that outline?

**Comment by mylesblaney** [comment]
light blue, it looks like some of the menus can be navigated by keyboard and have a light blue icon but a lot don't.

**Comment by abelardopardo** [comment]
Changed focus now to LightBlue
